src/sentry/integrations/github/client.py

```diff
@@ -224,6 +224,25 @@ def get_commit(self, repo: str, sha: str) -> Any:
         """
         return self.get_cached(f"/repos/{repo}/commits/{sha}")
 
+    def get_merge_commit_sha_from_commit(self, repo: str, sha: str) -> str | None:
+        """
+        Get the merge commit sha from a commit sha.
+        """
+        response = self.get_pullrequest_from_commit(repo, sha)
+        if not isinstance(response, list) or len(response) != 1:
+            # the response should return a single merged PR, return if multiple
```

Comment:
qq - when does this return multiple prs for the same commit?

Reply:
if there are multiple open PRs that contain the commit, like if somebody cherry picked the commit into different branches